.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+      - gh_actions
```

Comment:
A branch I created for playing with with gihub actions instead of master... I haven't found a way to run github actions locally (I tried ant but it did not work for me)

Reply:
Do you really want to keep it then? (i.e. are you done with testing it)

Reply:
Will remove it. Please note that I did not invest too much in ant it did not work for me out of the box so I left it for now...

Reply:
NP, was just wondering :)